tests/www/apply/test_submit.py

```diff
@@ -1956,6 +1955,9 @@ def test_apply_as_siae_with_suspension_sanction(self, client):
         )
 
     def _test_apply_as_company(self, client, user, company, dummy_job_seeker, pdf_file):
+        reset_url_dashboard = reverse("dashboard:index")
+        reset_url_company = reverse("companies_views:card", kwargs={"siae_id": company.pk})
+
```

Comment:
Çe donne plutôt envie de faire:
```suggestion.
        # Autoprescription: send to dashboard, otherwise send to other company card
        reset_url = reverse("dashboard:index") if company in user.company_set.all() else reverse("companies_views:card", kwargs={"siae_id": company.pk})

```
et d’utiliser `reset_url` plus loin.